web/server/codechecker_server/api/report_server.py

```diff
@@ -1998,37 +1994,20 @@ def getCheckerDoc(self, checkerId):
          - checkerId
         """
 
-        missing_doc = "No documentation found for checker: " + checkerId + \
-                      "\n\nPlease refer to the documentation at the "
+        return ""
 
-        if "." in checkerId:
-            sa_link = "http://clang-analyzer.llvm.org/available_checks.html"
-            missing_doc += "[ClangSA](" + sa_link + ")"
-        elif "-" in checkerId:
-            tidy_link = "http://clang.llvm.org/extra/clang-tidy/checks/" + \
-                checkerId + ".html"
-            missing_doc += "[ClangTidy](" + tidy_link + ")"
-        missing_doc += " homepage."
+    @exc_to_thrift_reqfail
+    @timeit
+    def getCheckerLabels(self, checkers):
```

Comment:
Can you please add a test case which tests this API function? Maybe we can put this extra test case here: https://github.com/Ericsson/codechecker/tree/master/web/tests/functional/report_viewer_api.